stubs/docutils/docutils/utils/__init__.pyi

```diff
@@ -20,16 +19,46 @@ class DependencyList:
 _SystemMessageLevel: TypeAlias = Literal[0, 1, 2, 3, 4]
 
 class Reporter:
+    levels: list[str]
+
     DEBUG_LEVEL: Literal[0]
     INFO_LEVEL: Literal[1]
     WARNING_LEVEL: Literal[2]
     ERROR_LEVEL: Literal[3]
     SEVERE_LEVEL: Literal[4]
 
+    def __init__(
+        self,
+        source: str,
+        report_level: int,
+        halt_level: int,
+        stream: IO[str] | str | bool | None = None,
+        debug: bool = False,
+        encoding: str | None = None,
+        error_handler: str = "backslashreplace",
+    ) -> None: ...
+
     source: str
+    error_handler: str
+    debug_flag: bool
     report_level: _SystemMessageLevel
-    halt_level: _SystemMessageLevel
-    def __getattr__(self, __name: str) -> Incomplete: ...
+    halt_level: int
+    stream: ErrorOutput
+    encoding: str
+    observers: list[Callable[[nodes.system_message], None]]
+    max_level: int
+    def set_conditions(
+        self, category: Any, report_level: int, halt_level: int, stream: IO[str] | None = None, debug: bool = False
+    ) -> None: ...
+    def attach_observer(self, observer: Callable[[nodes.system_message], None]) -> None: ...
+    def detach_observer(self, observer: Callable[[nodes.system_message], None]) -> None: ...
+    def notify_observers(self, message: nodes.system_message) -> None: ...
+    def system_message(self, level: int, message: str, *children: nodes.Node, **kwargs: Any) -> nodes.system_message: ...
+    def debug(self, *args: Any, **kwargs: Any) -> nodes.system_message: ...
+    def info(self, *args: Any, **kwargs: Any) -> nodes.system_message: ...
+    def warning(self, *args: Any, **kwargs: Any) -> nodes.system_message: ...
+    def error(self, *args: Any, **kwargs: Any) -> nodes.system_message: ...
+    def severe(self, *args: Any, **kwargs: Any) -> nodes.system_message: ...
```

Comment:
`args` should be `nodes.Node` per `system_message`'s annotations. See above for `kwargs` and `base_node`.

Reply:
```
+ sphinx/roles.py: note: In member "run" of class "PEP":
+ sphinx/roles.py:193:47: error: Argument 1 to "error" of "Reporter" has incompatible type "str"; expected "Node"  [arg-type]
+ sphinx/roles.py: note: In member "run" of class "RFC":
+ sphinx/roles.py:227:47: error: Argument 1 to "error" of "Reporter" has incompatible type "str"; expected "Node"  [arg-type]
+ sphinx/ext/ifconfig.py: note: In function "process_ifconfig_nodes":
+ sphinx/ext/ifconfig.py:66:46: error: Argument 1 to "error" of "Reporter" has incompatible type "str"; expected "Node"  [arg-type]
+ sphinx/ext/inheritance_diagram.py:382:52: error: Argument 1 to "warning" of "Reporter" has incompatible type "InheritanceException"; expected "Node"  [arg-type]
```

seems this is often called with a str, rather than a node